### PR TITLE
Add ColdCard instructions + PSBT

### PR DESCRIPTION
**Pull request by pavlenex**
This is just a part of already large PR #252 #241 

I'll add PSBT video soon, don't merge yet please, opened to feedback/review.

### CODE REVIEW

**Comment by pavlenex** [comment]
Ready to be reviewed!

File: ColdCardWallet.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+[![BTCPay and Coldcard](https://img.youtube.com/vi/N0eVwdP_7EQ/mqdefault.jpg)](https://www.youtube.com/watch?v=N0eVwdP_7EQ "Connecting Coldcard to BTCPay")
+
+###  Spending from BTCPay Server wallet with Coldcard (PSBT)
+
+Once there are some funds in received to your BTCPay Wallet connected to Coldcard, you can spend them by using [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md#psbt-in-general) (Partially Signed Bitcoin Transactions). This allows completely offline signing of your transactions, without ever having to connect your hardware wallet to the internet.
+
+1. Wallets > Manage > Send
+2. Fill in destination address and the amount
+3. Click on the Sign with `a wallet supporting PSBT` button.
+4. You will be redirected to PSBT tab, with pre-filled information, click on the  Sign with `a wallet supporting PSBT (save as file)`
```
Comment (britttttk) [inline ColdCardWallet.md]: (the  Sign) extra space
Also did you mean to include `Sign with` in the code formatted text line

File: ColdCardWallet.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+5. Save the file onto MicroSD card
+6. Insert MicroSD into your Coldcard.
+7. In Coldcard, click on the `Ready To Sign`
+8. Review the transaction information, and click OK button to sign it.
+9. Transaction will be saved onto MicroSD.
+10. In BTCPay, go to PSTB tab of the wallet and upload the signed PSBT file.
```
Comment (britttttk) [inline ColdCardWallet.md]: PSBT

File: ElectrumWallet.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 This documents shows how to connect Electrum Wallet to BTCPay Server. 
 
-**Word of caution** Using Electrum wallet relies on servers that are controlled by third-parties. Informations, like public adresses, balances and the transacted amount can potentially be leaked. To protect against such leaks, you can setup your own [Electrum Personnal Server](https://github.com/chris-belcher/electrum-personal-server), which is technically advanced task not covered in this document.
+**Word of caution** Using Electrum wallet relies on servers that are controlled by third-parties. Information's, like public addresses, balances and the transacted amount can potentially be leaked. To protect against such leaks, you can setup your own [Electrum Personnal Server](https://github.com/chris-belcher/electrum-personal-server), which is technically advanced task not covered in this document.
```
Comment (britttttk) [inline ElectrumWallet.md]: Information (is plural without the s)

File: ElectrumWallet.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 
 For a better privacy, use [BTCPay Wallet](Wallet.md) with a compatible hardware wallet or [Wassabi](WasabiWallet.md).
```
Comment (britttttk) [inline ElectrumWallet.md]: For better privacy

**Comment by pavlenex** [comment]
Typos fixed, thanks @britttttk for reviewing it.

File: ColdCardWallet.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+[![BTCPay and Coldcard](https://img.youtube.com/vi/N0eVwdP_7EQ/mqdefault.jpg)](https://www.youtube.com/watch?v=N0eVwdP_7EQ "Connecting Coldcard to BTCPay")
+
+###  Spending from BTCPay Server wallet with Coldcard (PSBT)
+
+Once there are some funds in received to your BTCPay Wallet connected to Coldcard, you can spend them by using [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md#psbt-in-general) (Partially Signed Bitcoin Transactions). This allows completely offline signing of your transactions, without ever having to connect your hardware wallet to the internet.
+
+1. Wallets > Manage > Send
+2. Fill in destination address and the amount
+3. Click on the Sign with `a wallet supporting PSBT` button.
+4. You will be redirected to PSBT tab, with pre-filled information, click on thec`Sign with a wallet supporting PSBT (save as file)`
```
Comment (britttttk) [inline ColdCardWallet.md]: the `Sign

Comment (pavlenex) [reply ColdCardWallet.md]: Dammit, I'm blind - done, thanks.

**Pull request merged**